about.html

```diff
@@ -0,0 +1,114 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Internal CSS -->
+    <link rel="stylesheet" href="./styles/about.css" />
+    <link rel="stylesheet" href="./styles/button.css" />
+    <!-- External CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <title>About</title>
+  </head>
+  <body>
+    <header>
+      <nav class="mobile-nav">
+        <i class="fa-solid fa-bars" id="hamburger"></i>
+      </nav>
+      <nav class="desktop-nav-2">
+        <ul class="flex-list">
+          <li><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
+          <li><a href=""></a><i class="fa-brands fa-twitter"></i></li>
+          <li><a href=""></a>English</li>
+          <li><a href=""></a>My Page</li>
+          <li><a href=""></a>Logout</li>
+        </ul>
+      </nav>
+      <nav class="desktop-nav">
+        <a href="index.html" class="nav-logo" ><img src="./images/logo-small.png" alt=""></a>
+        <ul class="flex-list close-menu">
+          <li><a href="about.html">About</a></li>
+          <li><a href="index.html">Register</a></li>
+          <li><a href="#about-school">Mission</a></li>
+          <li><a href="#testimonials">Facts</a></li>
+          <li class="last-child bubbly-button"><a href="#"></a>Killy Int'l</li>
+        </ul>
+      </nav>
+    </header>
+    <!-- Headline Section -->
+    <section id="headline-section">
+      <h1 class="intro-heading">LEARN THE PRACTICAL WAY</h1>
+
+      <div id="intro-text">
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tempore,
+          assumenda aperiam corporis nemo rem esse sed repellat expedita est
+          accusantium sit dignissimos, libero iste non quis dolorem sapiente
+          saepe autem.
+        </p>
+      </div>
+      <!-- <img src="./images/img2.png " alt="" /> -->
+      <div id="message-email">
+        <p>Please contact us via email for any questions or enquiries</p>
+        <h5 id="email">killynelly@gmail.com</h5>
+      </div>
+    </section>
+    <!-- About School Section -->
+    <section id="about-school">
+      <h2 id="about-heading">About School</h2>
+      <div class="about-intro">
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam iste
+          ducimus, officiis dicta eligendi ad beatae non itaque corrupti alias
+          natus quo saepe sit fugiat at repellendus consequatur. Corrupti, ex.
+        </p>
+      </div>
+      <div class="logo">
+        <img src="./images/school-logo.png" alt="" />
+      </div>
+    </section>
+    <!--Standout Section  -->
+    <section id="standout-section">
+      <h2 class="section-intro">Let Your Kid Standout</h2>
+      <hr class="orange-hr" />
+    </section>
```

Comment:
- [ Optional ] Well done on the cool animation effect in the `Register your child` button. And can we add the hover effect on the program card (Stats card in your case) by adding an `outline` as in the Design?
#### Design
![image](https://user-images.githubusercontent.com/44057863/169668914-5b3b435f-04ce-4c20-a8c3-fda115e6a5ad.png)
#### Impl
![image](https://user-images.githubusercontent.com/44057863/169668890-fdebb5de-c4e5-4594-b534-ec726abd4d8f.png)